### DISCUSSION TITLE
Summer holidays

### DISCUSSION THREAD

**Discussion by cweitkamp** [openhab/openhab-core#3499] [Core Maintainers Discussions]
Just to let you know: I'll be on vacation from 9th of June till 25th of June. On these days I won't have a computer at hand and plan to take a break from everything.

**Comment by maggu2810** [comment]
Thanks for the information.
Take your time and enjoy your vacation.

**Comment by kaikreuzer** [comment]
Enjoy your time off!
You'll be missed for a potential release preparations - but we might want to move those anyhow to July, let's see...

**Comment by wborn** [comment]
Have a great time off Christoph! Even if you don't touch openHAB, you'll still find out if it is stable and the Deconz OOM is fixed after your break. :wink:
